### PR TITLE
Issue #13809: kill mutation for RootNode getColumnNumber

### PR DESCRIPTION
**Pull request by romani**
Issue #13809

### CODE REVIEW

**Comment by romani** [comment]
@rnveach , do you remmber why lines starts with 1 but columns from 0 ?
Looks weird.

**Comment by romani** [comment]
@rnveach , I put link to issue as "until" for this pure unit test.

File: src/test/java/com/puppycrawl/tools/checkstyle/xpath/RootNodeTest.java [hunk 1]
```diff
@@ -110,6 +111,24 @@ public void testGetColumnNumber() {
             .isEqualTo(0);
     }
 
+    /**
+     * This test exists to cover pitest mutation.
+     * It is impossible to create RootNode that does not have column as 0.
+     * Test exists until https://github.com/checkstyle/checkstyle/issues/4997
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/xpath/RootNodeTest.java:117]: Inspections do not like raw links.
https://github.com/checkstyle/checkstyle/blob/c69d3aeb68da4215fc91bbf2e95b745b9f373994/src/test/java/com/puppycrawl/tools/checkstyle/XMLLoggerTest.java#L402

We should use non linked form of until.

Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/xpath/RootNodeTest.java:117]: Or keep it as non javadoc
https://github.com/checkstyle/checkstyle/blob/c69d3aeb68da4215fc91bbf2e95b745b9f373994/src/test/java/com/puppycrawl/tools/checkstyle/api/FileContentsTest.java#L275-L277

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**